### DISCUSSION TITLE
Change of Bitwarden releases location

### DISCUSSION THREAD

**Discussion by networkproblemsolver** [chocolatey/home#174] [General]
I'm not sure if this is the right place to post this, but I noticed "choco upgrade bitwarden" failed this morning.  It was trying to download from the /desktop/ location, but I found a note on the Bitwarden site that the new releases location is in their /clients/ area:  https://github.com/bitwarden/clients/releases

**Comment by TheCakeIsNaOH** [comment]
@networkproblemsolver the `bitwarden` package is maintained by the Bitwarden team themselves. You can see the source for the Chocolatey package here: https://github.com/bitwarden/clients/tree/master/apps/desktop/stores/chocolatey

I would suggest contacting them about your issue through either the "Contact Maintainer" form on the package page, or via any support methods you have with them (e.g. GitHub issues).
